src/averageAges.js

```diff
@@ -20,6 +20,22 @@ function calculateMenAverageAge(people, century) {
   // avoid using loop and forEach
   // replace `if ()` statement with &&, || or ?:
   // without nesting
+  // const MEN = people.filter(person => century
+  //   ? person.sex === 'm' && Math.ceil(person.died / 100) === century
+  //   : person.sex === 'm'
+  // );
+
+  const MEN = century
+    ? people.filter(person => person.sex === 'm'
+      && Math.ceil(person.died / 100) === century)
+    : people.filter(person => person.sex === 'm');
+
+  const MEN_AGES = MEN
```

Comment:
```suggestion
  const menAges = men
```
And in all other places throughout the code.